data.json

```diff
@@ -406,5 +406,12 @@
     "githubUsername": "DenverCoder1",
     "imageUrl": "https://user-images.githubusercontent.com/20955511/103463192-546c8100-4d33-11eb-9197-fec4c5396e27.png",
     "issueNumber": 209
+  },
+    {
```

Comment:
Could you adjust your formating the rest of the entries? Thanks 👍

Reply:
im sorry could you explain more?

Reply:
The indentation does not align consistently

Reply:
oh i see okay i'll try to fix it